### ISSUE TITLE
QGIS 3.28.0 Stuck Loading Projects With DXF Layers

### ISSUE THREAD

**Issue by Mozlo** [qgis/QGIS#50805]
### What is the bug or the crash?

Many projects that I work on will not load once opened. They will get to a certain % and then hang/not respond on a particular layer indefinately. 95% of the time these are DXF layers that have been imported. This is not a one off problem and occurrs constantly. Any info would be much appreciated.

### Steps to reproduce the issue

Opening .qgz file that is large/contains DXF layers

### Versions

3.28.0

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [X] I tried with a new QGIS profile

### Additional context

Resetting UI settings or creating a new profile does not fix this issue

**Comment by agiudiceandrea** [comment]
@Mozlo could you please attach a sample project with sample layer(s) that can be used to replicate the issue?

**Comment by Mozlo** [comment]
Hey mate, unfortunately the data I have is from clients and is confidential so I cannot post it at this point. I can say that it is not just DXF layers but also large statewide layers that it gets stuck on. This is acrosss multiple projects and multiple versions of QGIS (I have found that the later versions recently have done it more often than previous versions). I have been trying new versions almost monthly since Tisler (3.24.0). When opening on another PC sometimes they will open other times they will also get stuck. Would this potentially be a hardware problem? ie I need more RAM or something etc?

**Comment by Mozlo** [comment]
Have tried resetting UI settings as well as creating a new profile to try without addons. Does not fix the issue.

**Comment by Mozlo** [comment]
![Capture](https://user-images.githubusercontent.com/33919070/200450904-59aafbd5-0393-445d-ad99-3dadbb74c525.PNG)

**Comment by Mozlo** [comment]
Will just get stuck on a layer and then stop responding if you ever try to click back into QGIS. Am getting this problem every single day now. Do I need certain specs to be running QGIS consistantly?

**Comment by flykoh** [comment]
this problem happens to me and i didn't found any solve to it

**Comment by agiudiceandrea** [comment]
@flykoh, could you please attach a sample project with sample layer(s) that can be used to replicate the issue?
Does the issue also occur with previous versions of QGIS? What QGIS version are you using and on what operative system? Have you tried using a new QGIS user profile?

**Comment by flykoh** [comment]
yes, it occurs many times before i can't attach project file, because it contains sensitive database access credentials

**Comment by GGDRriedel** [comment]
I need to report this as well, trying to open a project saved with 3.24 though.
